config/src/tests/trace/trace.cpp

```diff
@@ -11,9 +10,9 @@ using namespace vespalib::slime;
 
 struct FixedClock : public Clock
 {
-    FixedClock() : currentTime(0) { }
-    int64_t currentTime;
-    int64_t currentTimeMillis() const override { return currentTime; }
+    FixedClock() : _currentTime(duration::zero()) { }
+    vespalib::system_time _currentTime;
```

Comment:
Outside the scope of this PR, but regarding the naming of `vespalib::system_time` I'm wondering if it would be semantically more obvious if we called it `vespalib::system_time_point`, as that's what it represents (as opposed to a _duration_ delta between two arbitrary time points)

Reply:
I am not sure. I suggest you discuss it with @havardpe.